components/Officers.js

```diff
@@ -137,7 +137,7 @@ const colors = [
 
 const Officers = () => {
 	return (
-		<div className='container flex flex-col items-center justify-center md:p-1 sm:p-1 mx-auto'>
+		<div className='container flex flex-col items-center justify-center md:p-1 sm:p-1 mx-auto mt-[130px]'>
```

Comment:
Where does 130px come from?

Reply:
Height of the div component that contains the navbar is around 126, so slightly more than that